packages/sdk/odahuflow/sdk/local/training.py

```diff
@@ -101,22 +124,27 @@ def start_train(trainer: K8sTrainer, output_dir: str):
     create_mt_config_file(trainer)
 
     if not output_dir:
-        # For example, 01-Mar-2020-17-38-04
-        suffix = datetime.datetime.now().strftime("%d-%b-%Y-%H-%M-%S")
-        output_dir = join(
-            config.LOCAL_MODEL_OUTPUT_DIR,
-            f'{trainer.model_training.spec.model.name}-{trainer.model_training.spec.model.version}-{suffix}'
-        )
+        output_dir = config.LOCAL_MODEL_OUTPUT_DIR
+        LOGGER.debug(f'Output directory for model training is not provided. Using default: {output_dir}')
 
-        LOGGER.debug(f'Output model training directory is not provided. Generate the directory: {output_dir}')
+    model_dir_name_template = trainer.model_training.spec.model.artifact_name_template or DEFAULT_MODEL_DIR_TEMPLATE
 
-    os.makedirs(output_dir, exist_ok=True)
+    model_dir_name_template = model_dir_name_template.rstrip('.zip')
```

Comment:
@BPylypenko From my point of view, if the artifactNameTemplate ends with `.zip` it tells us that user expects a zip-archive artifact. In case of local training the artifact is a directory. A directory name with .zip extension is valid but makes no sense. It is barely something that user meant. So I think it's worth to remove it.
In case of `wine-zapzip.zip` that you provided it will turn to `wine-zapzip`. Am I missing something?

Reply:
@keshamin Nope. As it's **rstip** function it will remove char by char from the end and we will finish with `wine-za` name of dir.

Reply:
Wow, you're right! I was pretty sure that rstrip works with the whole string, not a set of chars. Thanks!